programs/infrastructure/plan.md

```diff
@@ -5,9 +5,6 @@ title: Plan
 
 # Mainframe Infrastructure Plan for Open Mainframe Project
 
-* TOC
-{:toc}
-
```

Comment:
@lbdyck Can you revert this change? This adds the automatic TOC.

Reply:
I don't know how - can you accept it and then add that back in?